src/mono/wasm/runtime/driver.c

```diff
@@ -206,6 +199,24 @@ mono_wasm_add_assembly (const char *name, const unsigned char *data, unsigned in
 	return mono_has_pdb_checksum (data, size);
 }
 
+EMSCRIPTEN_KEEPALIVE void
+mono_wasm_mark_as_user_assembly (const char *assembly_name)
+{
+	WasmAssembly *entry = assemblies;
```

Comment:
Would be better to move this do mini-debugger-wasm.c, and simply keep the user assemblies in a hash.

Reply:
yeah a hash table in `mono-wasm-debugger.c` would cleaner.